### PR TITLE
Remove interactive questions for release.create-rc

### PR DESCRIPTION
**Pull request by CelianR**
### What does this PR do?

Removed interactive questions breaking the create-rc task. It removes the question from `deduce_version`.

### CODE REVIEW

**Review by amenasria — APPROVED** [review]
LGTM !

File: tasks/release.py [hunk 1]
```diff
@@ -84,20 +84,17 @@
...
 
-    if not os.isatty(sys.stdin.fileno()) or yes_no_question(
-        f'Version {release_version} deduced from branch {branch}. Is this the version you want to use?',
-        color="orange",
-        default=False,
-    ):
-        return release_version
+    print(
+        f'{color_message("Info", Color.BLUE)}: Version {release_version} deduced from branch {branch}', file=sys.stderr
+    )
```
Comment (chouquette) [inline tasks/release.py:95]: Wouldn't it be worth keeping the question if a TTY is attached?

**Review by chouquette — APPROVED** [review]
LGTM except for a non blocking question

**Review by pducolin — APPROVED** [review]

**Pull request merged**